noscrypto/Client.py

```diff
@@ -58,3 +58,13 @@ def WorldEncrypt(packet, session, is_first_packet = False):
     
 def WorldDecrypt(packet):
     return Utils._Unpack(packet, _DECRYPTION_TABLE)
+
+def WorldDecryptList(packets_raw, encoding="ascii"):

```

Comment:
I'd rather delete the `encoding` param, all of the existing functions return bytes, so I would rather be consistent and return array of raw bytes.